docs/settings.md

```diff
@@ -2,22 +2,22 @@
 Configuration for **Django RESTQL** is all namespaced inside a single Django setting named `RESTQL`, below is a list of what you can configure under `RESTQL` setting.
 
 ## QUERY_PARAM_NAME
-The default value for this is `query`. If you don't want to use the name `query` as your parameter, you can change it with`QUERY_PARAM_NAME` on settings file e.g 
+The default value for this is `query`. If you don't want to use the name `query` as your parameter, you can change it with`QUERY_PARAM_NAME` on settings file e.g
 ```py
 RESTQL = {
     'QUERY_PARAM_NAME': 'your_favourite_name'
 }
 ```
 Now you can use the name `your_favourite_name` as your query parameter. E.g
- 
+
 `GET /users/?your_favourite_name={id, username}`
 
 ## DEFAULT_BASE_FILTER_BACKEND
 The default value for this is `object`.
-This is used if you want to use query arguments to do filtering, this is discussed in detail on [Filtering with query arguments](#filtering-with-query-arguments)
+This is used if you want to use query arguments to do filtering, this is discussed in detail on [Filtering with query arguments](querying_data.md/#filtering-with-query-arguments)
```

Comment:
This gives 404 since the extension `.md` is not included in the URL, also it should start with the slash to prevent it from appending the URL to `settings/`. I think `/querying_data/#filtering-with-query-arguments` might work.